DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToHost/Raw/Geometry/PolylineConverterToHost.cs

```diff
@@ -0,0 +1,96 @@
+using Autodesk.Revit.DB;
+using Objects.Geometry;
+using Speckle.Converters.Common.Objects;
+using Speckle.Converters.RevitShared.Helpers;
+using Speckle.Converters.RevitShared.Services;
+using Speckle.Core.Logging;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+public class PolylineConverterToHost : ITypedConverter<SOG.Polyline, DB.CurveArray>
+{
+  private readonly ITypedConverter<SOG.Line, DB.Line> _lineConverter;
+  private readonly ScalingServiceToHost _scalingService;
+  private readonly IRevitConversionContextStack _contextStack;
+
+  public PolylineConverterToHost(
+    ITypedConverter<SOG.Line, DB.Line> lineConverter,
+    ScalingServiceToHost scalingService,
+    IRevitConversionContextStack contextStack
+  )
+  {
+    _lineConverter = lineConverter;
+    _scalingService = scalingService;
+    _contextStack = contextStack;
+  }
+
+  public CurveArray Convert(Polyline target)
+  {
+    var curveArray = new CurveArray();
+    if (target.value.Count == 6)
```

Comment:
guessing this was here before but could we comment why 6 is the magic number and this is correct just for future readers, i.e. if the Count (and that count is of) is 6 this means that blah blah...

Reply:
Can we have the comment?